systemtests/src/main/java/io/enmasse/systemtest/executor/client/AbstractClient.java

```diff
@@ -0,0 +1,184 @@
+package io.enmasse.systemtest.executor.client;
+
+import io.enmasse.systemtest.Logging;
+import io.enmasse.systemtest.executor.Executor;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.util.ArrayList;
+import java.util.concurrent.*;
+
+/**
+ * Class represent abstract client which keeps common features of client
+ */
+public abstract class AbstractClient {
+    private final int DEFAULT_ASYNC_TIMEOUT = 120000;
+    private final int DEFAULT_SYNC_TIMEOUT = 60000;
+
+    private ClientType clientType;

```

Comment:
clientType is private attribute without getter or setter and is used only like a first argument in arguments array, so it can be removed, I would say.

Reply:
fixed in 7222b744a71d46a2b7958b7cc6ad36a65aad3d5c